reactor-netty-http/build.gradle

```
@@ -151,6 +151,14 @@ dependencies {
 	testRuntimeOnly "org.slf4j:jcl-over-slf4j:$slf4jVersion"
 	testRuntimeOnly "ch.qos.logback:logback-classic:$logbackVersion"
 
+	// Needed for Brotli compression
+	testImplementation "com.aayushatharva.brotli4j:brotli4j:$brotli4jVersion"
+	testRuntimeOnly "com.aayushatharva.brotli4j:native-osx-aarch64:$brotli4jVersion"
+	testRuntimeOnly "com.aayushatharva.brotli4j:native-osx-x86_64:$brotli4jVersion"
+	testRuntimeOnly "com.aayushatharva.brotli4j:native-linux-x86_64:$brotli4jVersion"
+	testRuntimeOnly "com.aayushatharva.brotli4j:native-linux-aarch64:$brotli4jVersion"
+	testRuntimeOnly "com.aayushatharva.brotli4j:native-windows-x86_64:$brotli4jVersion"
```

Instead of importing the natives for all OSes, can we do something like this?


```suggestion
	if (osdetector.classifier in ["linux-aarch_64"] || ["osx-aarch_64"]) {
		testRuntimeOnly "com.aayushatharva.brotli4j:native-$osdetector.os-aarch64:$brotli4jVersion"
	}
	else {
		testRuntimeOnly "com.aayushatharva.brotli4j:native-$osdetector.classifier:$brotli4jVersion"
	}
```